internal/config.go

```diff
@@ -0,0 +1,86 @@
+package internal
+
+import (
+	"os"
+
+	"github.com/caarlos0/env"
+	"github.com/go-playground/validator"
+	"github.com/joho/godotenv"
+)
+
+var validate *validator.Validate
+
+const defaultPath = ".env"
+
+// IRCSettings includes settings related to the IRC bot/message relaying
+type IRCSettings struct {
+	Server              string   `env:"IRC_SERVER,required"`
+	Port                int      `env:"IRC_PORT" envDefault:"6667" validate:"min=0,max=65535"`
+	TLSAllowSelfSigned  bool     `env:"IRC_CERT_ALLOW_SELFSIGNED" envDefault:"true"`
+	TLSAllowCertExpired bool     `env:"IRC_CERT_ALLOW_EXPIRED" envDefault:"true"`
+	Channel             string   `env:"IRC_CHANNEL,required"`
+	BotName             string   `env:"IRC_BOT_NAME" envDefault:"teleirc"`
+	SendStickerEmoji    bool     `env:"IRC_SEND_STICKER_EMOJI" envDefault:"true"`
+	SendDocument        bool     `env:"IRC_SEND_DOCUMENT" envDefault:"true"`
+	Prefix              string   `env:"IRC_PREFIX" envDefault:"<"`
+	Suffix              string   `env:"IRC_SUFFIX" envDefault:">"`
+	ShowJoinMessage     bool     `env:"IRC_SHOW_JOIN_MESSAGE" envDefault:"true"`
+	ShowLeaveMessage    bool     `env:"IRC_SHOW_LEAVE_MESSAGE" envDefault:"true"`
+	NickServPassword    string   `env:"IRC_NICKSERV_PASS" envDefault:""`
+	NickServService     string   `env:"IRC_NICKSERV_SERVICE" envDefault:""`
+	EditedPrefix        string   `env:"IRC_EDITED_PREFIX" envDefault:"[EDIT]"`
```

Comment:
Super minor, but `"[EDIT] "` requires a space after the closing bracket

Reply:
Appreciated, thanks